mcproto/utils/abc.py

```diff
@@ -2,19 +2,21 @@
 
 from abc import ABC, abstractmethod
 from collections.abc import Sequence
-from typing import ClassVar
+from attrs import define
+from typing import Any, ClassVar
 
 from typing_extensions import Self
 
 from mcproto.buffer import Buffer
 
-__all__ = ["RequiredParamsABCMixin", "Serializable"]
+
+__all__ = ["RequiredParamsABCMixin", "Serializable", "define"]
```

Comment:
Don't re-export `attrs.define` here, there's no point in that. Also fix all of the imports that use `mcproto.utils.abc.define`, just import it from `attrs` like with any other import, we're not doing anything special with it anymore.